src/runtime/GlobalObjectBuiltinTypedArray.cpp

```diff
@@ -1607,6 +1607,26 @@ static Value builtinTypedArrayToStringTagGetter(ExecutionState& state, Value thi
     return Value();
 }
 
+static Value builtinTypedArrayAt(ExecutionState& state, Value thisValue, size_t argc, Value* argv, Optional<Object*> newTarget)
+{
+    // Let O be ? ToObject(this value).
+    if (!thisValue.isObject()) {
+        ErrorObject::throwBuiltinError(state, ErrorObject::TypeError, state.context()->staticStrings().ArrayIterator.string(), true, state.context()->staticStrings().next.string(), ErrorObject::Messages::GlobalObject_CalledOnIncompatibleReceiver);
+    }
+    Object* obj = thisValue.asObject();
+    validateTypedArray(state, obj, state.context()->staticStrings().TypedArray.string());
+    size_t len = obj->length(state);
+    double relativeStart = argv[0].toInteger(state);
+    if (relativeStart < 0) {
+        relativeStart = len + relativeStart;
+    }
+    if (relativeStart < 0 || relativeStart >= len) {
+        return Value();
+    }
+
+    return obj->get(state, ObjectPropertyName(state, Value(relativeStart).toString(state))).value(state, thisValue);
```

Comment:
I belive we can remove toString operation.
just passing number into ObjectPropertyName(state, Value(relativeStart))